files/zh-cn/learn/javascript/client-side_web_apis/client-side_storage/index.md

```diff
@@ -491,7 +491,7 @@ function displayData() {
 
 再次，让我们打破这个：
 
-- 首先，我们清空[ 元素 ( 或 HTML 无序列表元素） 代表多项的无序列表，即无数值排序项的集合，且它们在列表中的顺序是没有意义的。通常情况下，无序列表项的头部可以是几种形式，如一个点，一个圆形或方形。头部的风格并不是在页面的 HTML 描述定义, 但在其相关的 CSS 可以用 list-style-type 属性。">`<ul>`](/zh-CN/docs/Web/HTML/Element/ul)元素的内容，然后填充更新的内容。如果您不这样做，那么每次更新时都会添加大量重复内容。
+- 首先，我们清空 [元素 ( 或 HTML 无序列表元素） 代表多项的无序列表，即无数值排序项的集合，且它们在列表中的顺序是没有意义的。通常情况下，无序列表项的头部可以是几种形式，如一个点，一个圆形或方形。头部的风格并不是在页面的 HTML 描述定义, 但在其相关的 CSS 可以用 list-style-type 属性。">`<ul>`](/zh-CN/docs/Web/HTML/Element/ul)元素的内容，然后填充更新的内容。如果您不这样做，那么每次更新时都会添加大量重复内容。
```

Comment:
I think we can update those words:

```suggestion
- 首先，我们清空 [`<ul>`](/zh-CN/docs/Web/HTML/Element/ul) 元素的内容，然后填充更新的内容。如果您不这样做，那么每次更新时都会添加大量重复内容。
```

![image](https://user-images.githubusercontent.com/15844309/184272194-c65dec4b-b8c6-4b8e-b498-68fd93d1bef8.png)